automation/check-patch.mounts

```diff
@@ -3,3 +3,7 @@
 /var/cache:/var/host_cache
 /var/run/docker.sock:/var/run/docker.sock
 /var/lib/stdci/shared:/var/lib/stdci/shared
+/var/run/docker/netns:/var/run/docker/netns
```

Comment:
Ouch, I should not touch the default file but instead create the job specific one...